backend/sokdak/src/test/java/com/wooteco/sokdak/member/acceptance/MemberAcceptanceTest.java

```diff
@@ -1,20 +1,70 @@
 package com.wooteco.sokdak.member.acceptance;
 
+import static com.wooteco.sokdak.util.fixture.HttpMethodFixture.httpGetWithAuthorization;
+import static com.wooteco.sokdak.util.fixture.HttpMethodFixture.httpPatchWithAuthorization;
+import static com.wooteco.sokdak.util.fixture.HttpMethodFixture.httpPost;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+
+import com.wooteco.sokdak.auth.dto.LoginRequest;
+import com.wooteco.sokdak.member.dto.NicknameResponse;
+import com.wooteco.sokdak.member.dto.NicknameUpdateRequest;
 import com.wooteco.sokdak.util.AcceptanceTest;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
 import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
 
 @DisplayName("회원 관련 인수테스트")
 class MemberAcceptanceTest extends AcceptanceTest {
-//    @DisplayName("이메일로 우테코 회원임을 인즈")
-//    @Test
-//    void send() {
-////        이메일 인증
-//        EmailRequest emailRequest = new EmailRequest("sokdak@gmail.com");
-//        ExtractableResponse<Response> response = httpPost(emailRequest, "/signup/email");
-//        assertThat(response.statusCode()).isEqualTo(HttpStatus.NO_CONTENT.value());
-//    }
-    //        인증번호 확인
-    // TODO:
+
+//     TODO:
 //        아이디 중복 확인
 //        회원가입 완료
+
+    @DisplayName("닉네임을 수정할 수 있다.")
+    @Test
+    void editNickname() {
+        String nickname = "chrisNick2";
+        NicknameUpdateRequest nicknameRequest = new NicknameUpdateRequest(nickname);
+        String token = getToken();
+
+        ExtractableResponse<Response> response = httpPatchWithAuthorization(nicknameRequest,
+                "/members/nickname", token);
```

Comment:
```suggestion
        ExtractableResponse<Response> response = 
                httpPatchWithAuthorization(nicknameRequest, "/members/nickname", token);
```
저는 위와 같은 개행이 더 읽기 좋은 것 같아요! 고려해보셔도 좋을 것 같아요~!